src/log.js

```diff
@@ -32,7 +31,11 @@ module.exports = {
 		read(options, callback);
 	},
 
-	getHumanDate(ts) {
-		return moment(ts).format("YYYY-MM-DD HH:mm:ss");
+	getHumanDate(date) {
+		if (!date) {
+			date = new Date();
+		}
```

Comment:
Alternatively, we could do `return new Date(date).toISOString()`, as `new Date()`ing a date still returns a similar date (extra object, but whatever IMO).